src/app/main.ts

```diff
@@ -2,6 +2,7 @@ import App from "./App.svelte";
 
 const app = new App({
   target: document.body,
+  hydrate: true,
```

Comment:
If `server.js` is not necessary, then this line is not necessary. As far as I understand, hydratable components only make sense if we have server-side rendering, which we don't! 😆 

```suggestion
```

Reply:
Same as above, this `hydrate: true` is needed to make the routing work as per the documentation from the author